src/griddyn/loads/aggregateLoad.cpp

```diff
@@ -189,11 +191,9 @@ void aggregateLoad::dynObjectInitializeA (coreTime time0, std::uint32_t flags)
 
 void aggregateLoad::dynObjectInitializeB (const IOdata &inputs, const IOdata &desiredOutput, IOdata &fieldSet)
 {
-    IOdata out = desiredOutput;
-
     for (auto &ld : subLoads)
     {
-        ld->dynInitializeB (inputs, out, fieldSet);
+        ld->dynInitializeB (inputs, desiredOutput, fieldSet);
```

Comment:
Add a note here that more needs to be done in case desiredOutput actually contains something of use.  I believe all examples use it as empty so this won't matter, but eventually something needs to be one.  So we should add a TODO here